### PR TITLE
data: joelnyongesa.json

### PR DESCRIPTION
**Pull request by joelnyongesa**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/joelnyongesa.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      "name": "GitHub: Follow me",
+      "url": "https://github.com/joelnyongesa",
+      "icon": "FaGithub",
+      "color": "#00ACEE"
+    },
+    {
+      "group": "Socials Primary",
+      "name": "LinkedIn: Lets Connect",
+      "url": "https://linkedin.com/in/jknyongesa",
+      "icon": "FaLinkedin"
```
Comment (sital002) [inline data/joelnyongesa.json:45]:
```suggestion
      "icon": "FaLinkedin",
```
There is a comma missing here

**Review by sital002 — DISMISSED** [review]
Please make the necessary changes to get your profile merged

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for Contributing! 🎉

**Comment by joelnyongesa** [comment]
Thanks @kumarsonsoff3 . I appreciate your help. When I go to biodrop.io/joelnyongesa, I can't find my profile. Is this so on your end as well?

**Comment by kumarsonsoff3** [comment]
> Is this so on your end as well?

Nope, it should be active now. Please try again at: https://www.biodrop.io/joelnyongesa

**Comment by joelnyongesa** [comment]
> > Is this so on your end as well?
> 
> Nope, it should be active now. Please try again at: https://www.biodrop.io/joelnyongesa

Thanks so much @kumarsonsoff3

**Pull request merged**